packages/integration-sdk-core/jest.config.js

```diff
@@ -1,4 +1,4 @@
 module.exports = {
   ...require('../../jest.config.base'),
-  setupFilesAfterEnv: ['./jest.setup.ts'],
+  //setupFilesAfterEnv: ['./jest.setup.ts'],
```

Comment:
why is this line commented?

Reply:
Pre commit hook runs jest and jest tries to pickup that file but it doesn't exist.